konduit-serving-cli/src/main/resources/META-INF/konduit-serving-cli-git.properties

```diff
@@ -0,0 +1,20 @@
+#Generated by Git-Commit-Id-Plugin
```

Comment:
These files are generated during the build, so we probably don't need to commit them to the repo.
That may mean changing the config to output /target/classes/META-INF/... though, instead of /src/main/resources/META-INF/...